Añadir requerimientos minímos al readme.md

Veo que nuestro stack necesita unos minimos 

- Go v1.10.0+
- Docker-compose v18.06.0+

por ejemplo al hacer make build sin satisfacer la versión para tener el `compose file format`

```shell
docker-compose -f ./docker/docker-compose.yml build
ERROR: Version in "././docker/docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
make: *** [Makefile:3: build] Error 1
```


Añadir estos minimos en la documentación del proyecto

--- a/README.md
+++ b/README.md
@@ -5,6 +5,11 @@ Su objetivo es posicionar en un mapa los Almeces existentes en la ciudad.
 
 ## Desarrollo
 
+Requisitos para el desarrollo:
+
+- Golang v1.12.7, o superior
+- Docker Compose v1.24.1, o superior
+
 El lenguaje utilizado para crear el API será Golang, utilizando `Go modules`. De este modo es necesario que la variable de entorno `GO111MODULE` está activada con un valor `on`.
 
 ```shell
@@ -17,7 +22,7 @@ $ export GO111MODULE=on && code /path/al/proyecto
 
 ## Ejecución
 
-El proyecto usa `Make` como wrapper para lanzar los comandos, y deberías lanzarlos en el siguiente orden para tener el proyecto corriendo en un estado adecuado:
+El proyecto usa `Make` como wrapper para lanzar los comandos, que usa internamente `Docker Compose` para levantar el stack en local. Deberías lanzar los comandos en el siguiente orden para tener el proyecto corriendo en un estado adecuado:
 
 ```shell
 $ make build   # construye la imagen Docker con el binario de Go

